dataland-frontend/src/components/resources/dataTable/conversion/MultiLayerDataTableFieldConverter.ts

```diff
@@ -71,3 +77,12 @@ export function getDataModelFieldCellConfig(path: string, field: Field): MLDTCel
     return undefined;
   }
 }
+
+/**
+ * Checks if fields with null values should be shown or not
+ * @param value This is the displayValue parsed from the field config
+ * @returns boolean to set hidden to true or false
+ */
+function checkToShowFieldsWithNullValue(value: MLDTDisplayComponentTypes[MLDTDisplayComponentName]): boolean {
```

Comment:
This function name suggest usage of the function. I would prefer something like `shouldValueBeDisplayed`.

Reply:
Changed